_transformations/scale/readme.md

```diff
@@ -14,17 +14,43 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges in a scale file.
+The input string must be in numerical format.
 
 The file is expected to exist in the `transform` directory, and should follow the format given in the example below.
 
 ## Example
 
-transform/temps.scale:
+The following example shows how to break down numeric UV values into fixed UV index categories.
 
-```properties
-[12..23]=Temp between 12 or higher and 23 or lower
-]12..23[=Temp over 12 and below 23
-..23]=Any temp up to and including 23
-]12..=Any temp above 12
+example item:
+
+```java
+Number Uv_Sensor_Level "UV Level [SCALE(uvindex.scale):%s]"
+```
+
+uvindex.scale in transform folder:
+
+```
```

Comment:
Every code block should have a language. See: http://www.cirosantilli.com/markdown-style-guide/#option-code-fenced

Will also look nicer:
```python
[0,3]=1
[3,6]=2
[6,8]=3
[8,10]=4
[10,100]=5
```

Reply:
Same procedure as every time...
I copied that from the wiki and didn't check it. :D

Which language did you choose for that?

Reply:
I chose python, simply because I know that *lists* are defined with square brackets.

Python:
```python
[0,3]=1
[3,6]=2
[6,8]=3
[8,10]=4
[10,100]=5
```

Java:
```java
[0,3]=1
[3,6]=2
[6,8]=3
[8,10]=4
[10,100]=5
```

PHP:
```php
[0,3]=1
[3,6]=2
[6,8]=3
[8,10]=4
[10,100]=5
```

Yeah... I think it doesn't really matter ;)